roles/common/tasks/main.yml

```diff
@@ -18,6 +18,10 @@
   tags:
     - common
 
+- name: ensure cloud.cfg.d directory exists
+  sudo: yes
+  file: path=/etc/cloud/cloud.cfg.d state=directory recurse=yes owner=root group=root mode=0644
```

Comment:
Yeah... that was the reason.  Wasn't sure what the purpose of the preserve hostname task was so just added the directory so it would pass... I can switch to excluding it if that's a better option.